types/firestore.d.ts

```diff
@@ -903,6 +903,21 @@ declare namespace FirebaseFirestore {
      * @return The created Query.
      */
     limit(limit: number): Query<T>;
+    
+    /**
+     * Creates and returns a new Query that only returns the last matching
```

Comment:
Same comment about bringing `limit()`'s comment in the same wording.

Reply:
Also done here.